updateRepo.sh

```diff
@@ -1,34 +1,53 @@
+#!/bin/bash
+
 echo "Checking if repo needs to be updated"
 export ORG="IBM-Swift"
 export REPO="generator-swiftserver-projects"
 export GH_REPO="github.com/${ORG}/${REPO}.git"
-export BRANCH="init"
+export BRANCHES="init openAPI"
 export projectName="Generator-Swiftserver-Projects"
 
-cd ${TRAVIS_BUILD_DIR}
-mkdir current
-cd current
-git clone -b ${BRANCH} "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${ORG}/${REPO}.git"
-export currentProject=`pwd`
+for BRANCH in $BRANCHES
+do
+  cd "${TRAVIS_BUILD_DIR}" || exit
+  mkdir current
+  cd current || exit
+  git clone -b "${BRANCH}" "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${ORG}/${REPO}.git"
+  currentProject=$(pwd)
+
+  mkdir -p "${TRAVIS_BUILD_DIR}/new/${projectName}"
```

Comment:
Why are we creating TRAVIS_BUILD_DIR, aren't we already in that directory?

Reply:
This relates to work prior to my PR so I'm not 100% sure, previously it used `../` I felt this was a better approach. 
At this stage of the script though we are in the following directory: `${TRAVIS_BUILD_DIR}/current`

Reply:
I'm confused.  On line 12 we `cd` into `${TRAVIS_BUILD_DIR}` but on line 18 we create it?

Reply:
Ah I understand now. 

The `TRAVIS_BUILD_DIR` isn't something we've created, it's an environment variable for Travis. We just use it to ensure we're creating sub directories in a known location.

Reply:
But this line will create a subdirectory tree with that name, won't it?

Reply:
`mkdir` works relative to the current directory, it's not absolute.

Reply:
Or is `${TRAVIS_BUILD_DIR}` absolute? If so, we don't need to specify it here do we?

Reply:
I'm not entirely sure what you mean here. 

The command in question: 
`mkdir -p "${TRAVIS_BUILD_DIR}/new/${projectName}"`

Creates the `new/${projectName}"` directory within the `${TRAVIS_BUILD_DIR}` directory. Which is what the original script was intending to do, but the original script just used `../` instead. It does not recreate the `${TRAVIS_BUILD_DIR}` within itself. 

I've tested this in the [following build](https://travis-ci.org/ddunn2/generator-swiftserver-projects/builds/479281458#L1098) where after creating the directory I attempt to `cd` into the `new` directory to ensure it has been created in the correct location. **Note:** I have to step up one level as we are still inside the `current` directory.